st2common/st2common/operators.py

```diff
@@ -215,6 +223,10 @@ def nexists(value, criteria_pattern):
 TIMEDIFF_GT_LONG = 'timediff_gt'
 KEY_EXISTS = 'exists'
 KEY_NOT_EXISTS = 'nexists'
+INSIDE_LONG = 'inside'
+INSIDE_SHORT = 'in'
```

Comment:
This shortcut may be misleading... Or not. Dunno.

Reply:
I think this shortcut makes since as it is semantically the same as a python `in` check. I.e., `trigger in provided_value`.